mailu/templates/front/service-external.yaml

```diff
@@ -19,6 +19,9 @@ spec:
 {{- with .Values.front.externalService }}
   type: {{ .type | default "ClusterIP" }}
   externalTrafficPolicy: {{ .externalTrafficPolicy | default "Local" }}
+  {{- with .ipFamilyPolicy }}
```

Comment:
Why use `with` here and not `if`? 
It would seem that it's not certain that `ipFamilyPolicy` exists.
`with` assumes your variable is present: https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with

Reply:
With and if should behave the same if the variable is absent (not yielding anything), but `with` only makes you write the name of the field (`ipFamilyPolicy`) once. So the chance to make a typo is reduced by 50% 😁

Reply:
The "typo chance" is not a joke, I think the vast majority of helm chart bugs I've seen, fixed, and introduced come from copypasting snippets and forget to change part of them.